config/dtm/test_dtm_config.yaml

```diff
@@ -1,23 +1,18 @@
-clients: 1
-samples: 500
+clients: 10
+samples: 1000
 size: '128Mb'
 loop_count: 10
+delay: 30 #Before service restart
+
 #sizes in size_list are in Mb
 size_list: [1, 2, 3, 4, 5, 6, 7]
 # count to restart the rgw process
 rgw_restart_cnt: 5
```

Comment:
Same as above.

Reply:
Anything that is used in test is kept as part of test_dtm_config.